app/views/placements/wizards/add_mentor_wizard/_check_your_answers_step.html.erb

```diff
@@ -38,9 +39,8 @@
         <p class="govuk-inset-text">
           <%= embedded_link_text(
                 t(".confirm_mentor_informed",
-                  mentor_name: @wizard.steps[:mentor].mentor.full_name,
                   link: govuk_link_to(
-                    t(".privacy_notice"), placements_privacy_path, no_visited_state: true
+                    t(".privacy_notice"), placements_privacy_path, target: "_blank", new_tab: true, no_visited_state: true
```

Comment:
```suggestion
                    t(".privacy_notice"), placements_privacy_path, new_tab: true, no_visited_state: true
```

The `new_tab` argument sets `target="_blank"` for us automatically, but unfortunately this isn't working on the review app, it looks like the `embedded_link_text` is eating our `target="_blank"` 🤔

Reply:
Is the `target="_blank"` needed here?